test/caches.test.js

```diff
@@ -32,7 +32,7 @@ beforeAll(() => {
       case absolutePath('a/b/c/foo.config.js'):
         throw { code: 'ENOENT' };
       case absolutePath('a/b/package.json'):
-        return '{ "foundInB": true }';
+        return '{ "foo": { "foundInB": true } }';
```

Comment:
Why did this test need to change?

Reply:
Because this test checks the contents of the configuration, which has changed.

https://github.com/azz/cosmiconfig/blob/ce99ddee32b71658a5d08b02778fe2430ce8bdb3/test/caches.test.js#L140-L150